spdockit-spo/spo-snapshots.md

```diff
@@ -38,5 +40,22 @@ Here is what you need to do:
 
 1. Click Next and the loading will start. Wait for the SPDocKit wizard to finish, then SharePoint [farm settings](#internal/get-to-know-spdockit/farm-explorer-screen/farm-explorer-reports) and [permission reports](#internal/get-to-know-spdockit/permissions-reports-screen) will be ready for use!  
 
-    Also, make sure to check our [permission management features](#internal/permission-management/manage-permissions-ribbon-actions) located in __Permissions Explorer__ tab.  
-    Please note that Permission management buttons are enabled only in __live mode__ of Permissions Explorer. There is no need for a snapshot in this case. 
+
+## Automatic Snapshot
+
+SPDocKit Service allows you to configure a schedule to create automatic SharePoint Online snapshots. Once configured, the SPDocKit Service can save your farm settings and send report subscriptions and alerts to selected users, document libraries, or network locations. Click [here](internal/how-to/subscriptions-and-alerts/create-new-subscription) for more information on how to do this.  
+
+It is not required to have Local Admin privilages for a service account. It can be any account, even Local System. See [here](https://msdn.microsoft.com/en-us/library/windows/desktop/ms684190(v=vs.85).aspx) for more information on the Local System account. While setting up the service account via configuration wizard, simply choose what is more convenient to you. 
```

Comment:
Since we are dealing with a workstation install, to NOT have Local Admin is actually expected from the user. So no reason to mention it at all. The local system additional info link is also a bit superfluous since it's content is a bit hardcore.

suggestion:
_The service account can be any account, including the Local System Account._

Reply:
Seconding this suggestion, looks to be much more easier to understand. Also, if you choose to keep the contents of the paragraph, it's privileges.